docs/tools/create-storage-class.sh

```diff
@@ -82,7 +82,7 @@ metadata:
     directpv.min.io/version: v1beta1
   name: ${NAME}
 parameters:
-  fstype: xfs
+  csi.storage.k8s.io/fstype: xfs
```

Comment:
Will this be a harmless change?

Reply:
yes, this is harmless. 

I have seen logs like `"fstype" is deprecated and will be removed in a future release, please use "csi.storage.k8s.io/fstype" instead` in older k8s versions

Reply:
okay wonderful.